.github/ISSUE_TEMPLATE.md

```diff
@@ -7,7 +7,12 @@ please take a look at [our contribution guide](https://docs.chainer.org/en/stabl
 Specifically, if it is a bug report, these information are very helpful:
 
 * Conditions
-<!-- If you're using Chainer 4.0+, you can also get this information by typing `python -c 'import chainer; chainer.print_runtime_info()'. -->
+<!--
+If you're using Chainer 4.0+, you can also get this information by typing the following:
```

Comment:
Let's remove `If you're using Chainer 4.0+, ` part, as we no longer support Chainer v3.